### PR TITLE
Added backend setup info to README

### PR DESCRIPTION
**Pull request by felixlapierre**
# General info

**Issue number**: N/A

**Task description**: 
- Added setup information for the backend, which was missing in the current Readme. Also, added some information for resolving common issues with setting up the app. I also copied the relevant information about the google java formatter from their README into ours, to make it quicker for people to set up backend formatting.
 
**Test coverage**: N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,22 +27,53 @@ Developed as a Concordia University Capstone project.
...
+    * Download and install MySQL Community 8.0. Make sure you install at minimum the server and MySQL Workbench.
+    * Take note of your root user password. 
+        * If you were not prompted for a password during installation, go to the bin folder of your MySQL installation and run the command `mysqld --initialize --console`. This should initialize the database and print a randomly-generated one-time password to the console.
+    * Open MySQL workbench and connect to your database.
+    * Create a new schema named `switchboard`. If you ever need to reset the contents of your database, delete this schema and re-create it.    * You must create a schema named `switchboard` for use with the application.
+    
+#### Running
+* `./mvnw install` to install dependencies.
+* `./mvnw compile` to compile the code.
+* `./mvnw spring-boot:run` to run the application.
```
Comment (mohhef) [inline README.md:53]: maybe we can mention to clean first incase you want to recompile

File: README.md [hunk 2]
```diff
@@ -27,22 +27,53 @@ Developed as a Concordia University Capstone project.
...
+#### Setup
+* Install Java 11 SDK
+    * To check that you are using the correct version:
+        * In your terminal, run `java -version`
+        * In IntelliJ, you can check File → Project Structure → Project SDK
+        
+* Install MySQL 8.0
+    * Download and install MySQL Community 8.0. Make sure you install at minimum the server and MySQL Workbench.
+    * Take note of your root user password. 
+        * If you were not prompted for a password during installation, go to the bin folder of your MySQL installation and run the command `mysqld --initialize --console`. This should initialize the database and print a randomly-generated one-time password to the console.
```
Comment (mohhef) [inline README.md:46]: maybe we should mention the mysql port we are using -> 3306, its pretty standard but just in incase

**Review by mohhef — DISMISSED** [review]
Looks good to me

File: README.md (hunk 2 above)
Comment (ButlerJAL) [reply README.md:46]: Personal preference, I would not specify configuration port numbers in the README, especially not default ones

**Review by ButlerJAL — APPROVED** [review]
LGTM

**Review by mohhef — APPROVED** [review]
Looks good

**Pull request merged**